### ISSUE TITLE
Feature : Layer 2 support ipv6 

### ISSUE THREAD

**Issue by chaunceyjiang** [openelb/openelb#229]
Hi, is this feature being implemented? if not , i want to try to implement this feature.
As far as I know,
IPv4 use ARP ,IPv6 use [NDP](https://en.wikipedia.org/wiki/Neighbor_Discovery_Protocol)

Reference
https://en.wikipedia.org/wiki/Neighbor_Discovery_Protocol
https://github.com/openelb/openelb/blob/master/doc/roadmap.md#v070

**Comment by FeynmanZhou** [comment]
> 

Hi @chaunceyjiang ,

It was listed in the roadmap but it was also pending for so long. We are glad to see your interest in contributing to this feature. You can assign it to yourself if you want.

**Comment by viceice** [comment]
- related to #154

**Comment by FeynmanZhou** [comment]
> * related to [support ipv6 in bgp & layer2 mode #154](https://github.com/openelb/openelb/issues/154)

Yes, we could close that issue since it is not active anymore. We can track it via this issue from now on.

**Comment by chaunceyjiang** [comment]
https://github.com/kubernetes/kubernetes/pull/107235
https://github.com/kubernetes/enhancements/pull/1992
